### PR TITLE
Combined dependabot security upgrades

### PR DESCRIPTION
**Pull request by thomas11**

### CODE REVIEW

**Review by rquitales — APPROVED** [review]

File: sdk/go.mod
Comment (danielrbradley) [inline sdk/go.mod:1]: Oh this removal is interesting - should we just remove this mod now that we publish this elsewhere?

Comment (thomas11) [reply sdk/go.mod:1]: Oh, TBH I didn't even notice. I just ran `go mod tidy` and checked that there was no error. I guess we can remove it.

Comment (thomas11) [reply sdk/go.mod:1]: Oh, TBH I hadn't even noticed. I just ran `go mod tidy`. I guess we can remove it.

Comment (danielrbradley) [reply sdk/go.mod:1]: Don't know if it's worth leaving some kind of stub there so people will be directed to the new location ... but might not need the mod for that.

**Pull request merged**